audeer/core/conftest.py

```diff
@@ -0,0 +1,9 @@
+import pytest
+
+
+@pytest.fixture(autouse=True)
+def docdir(request):
+    # set temporal working directory in docstring examples
+    tmpdir = request.getfixturevalue('tmpdir')
+    with tmpdir.as_cwd():
```

Comment:
Cool, that's nice.
So far I always relied on cleaning up the root folder of the repo afterwards. This seems to be more elegant.